ccm_web/client/src/pages/GradebookCanvas.tsx

```diff
@@ -98,7 +99,7 @@ const convertEmptyCellToUndefined = (cell: string | undefined): string | undefin
 }
 
 function ConvertCanvasGradebook (props: CCMComponentProps): JSX.Element {
-  const [pageState, setPageState] = useState<GradebookCanvasPageStateData>({ state: GradebookCanvasPageState.Upload })
+  const [pageState, setPageState] = useState<GradebookCanvasPageStateData>({ state: GradebookCanvasPageState.ReplacementNotice })
   const [file, setFile] = useState<File|undefined>(undefined)
```

Comment:
I would say don't touch this component for adding this Text notification about retirement, create a new component saying "GBCanvasReplacementNotice` something  like that, and wire this component instead of GradebookCanvas.tsx. So the logic for displaying title, who has access to this component still remains same. Just replace it with new component.

Reply:
I mean to say new page, and add this page to the list of display in `FeatureUIData.tsx` , Don't change the URL simply replace with new Componenet


```
const formatCanvasGradebookCardProps: FeatureUIProps = {
  data: formatCanvasGradebookProps,
  icon: <LibraryBooksOutlinedIcon fontSize='large' />,
  component: ConvertCanvasGradebook,
  route: '/gradebook-canvas'
}
```